lib/measured/base.rb

```diff
@@ -27,6 +36,10 @@ def method_missing(method, *args)
 require "measured/arithmetic"
 require "measured/unit"
 require "measured/conversion"
+require "measured/conversion_builder"
 require "measured/conversion_table"
 require "measured/measurable"
-require "measured/case_sensitive_measurable"
+  
+module Measured
+  private_constant :Measurable
```

Comment:
We are better off leaving it to be subclassed, but having it fail to work without being called from the `build` interface. Like, raise if you have a bare subclass that hasn't been setup, or if you try to mutate it.